lib/chef/mixin/params_validate.rb

```diff
@@ -174,7 +174,9 @@ def _pv_equal_to(opts, key, to_be)
           to_be.each do |tb|
             return true if value == tb
           end
-          raise Exceptions::ValidationFailed, _validation_message(key, "Option #{key} must be equal to one of: #{to_be.join(", ")}!  You passed #{value.inspect}.")
+          # Ruby will print :something as something, which confuses users so make sure to print them as symbols
+          corrected_type_array = to_be.collect { |x| x.kind_of?(Symbol) ? ":#{x}" : x }
+          raise Exceptions::ValidationFailed, _validation_message(key, "Option #{key} must be equal to one of: #{corrected_type_array.join(", ")}!  You passed #{value.inspect}.")
```

Comment:
`#{to_be.map { |v| v.inspect }.join(", ")}` is probably a better way to fix that

```ruby
% pry
[1] pry(main)> :symbol.inspect
=> ":symbol"
```